databases/week1/queries-hw1.sql

```diff
@@ -0,0 +1,84 @@
+/*Find out how many tasks are in the task table*/
+SELECT
+  COUNT(*)
+FROM
+  task;
```

Comment:
for readability it's nicer to have the query and statement on the same line. e.g.
`SELECT COUNT(*)`
`From task`

This applies to the other parts as well :)